<ui version="4.0" >
 <class>INDIHostConf</class>
 <widget class="QDialog" name="INDIHostConf" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>178</height>
   </rect>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_3" >
         <property name="text" >
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Port:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="KLineEdit" name="nameIN" />
       </item>
       <item>
        <widget class="KLineEdit" name="hostname" />
       </item>
       <item>
        <widget class="KLineEdit" name="portnumber" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>30</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameIN</tabstop>
  <tabstop>hostname</tabstop>
  <tabstop>portnumber</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>INDIHostConf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>288</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>162</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>INDIHostConf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>386</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel" >
     <x>386</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
